rasa/core/test.py

```diff
@@ -619,11 +627,20 @@ def _collect_action_executed_predictions(
             # that something else than the form was supposed to run.
             predicted = action.name()
 
-    action_executed_eval_store.add_to_store(
-        action_predictions=[predicted], action_targets=[gold]
+    should_ignore_prediction = action_executed_eval_store.should_ignore_prediction(
+        predicted
     )
+    if not should_ignore_prediction:
+        action_executed_eval_store.add_to_store(
+            action_predictions=[predicted], action_targets=[gold]
```

Comment:
This might be a silly q - what does `gold` mean in this context - I can see it's defined earlier, but I don't grasp what it means?

Reply:
It's the action that's specified in the test story. I guess it's `gold` because we treat as the expected action, something like the "golden standard" or whatever.
Even though it was defined earlier, I've actually just replaced it with `expected_action` just because it makes a bit more sense in this context.